### PR TITLE
Generate tests in the test workflow

### PR DESCRIPTION
**Pull request by cgccuser**
I just realized I made a mistake in elements.yml but it wasn't noticed because the tests weren't regenerated, which is not good.

### CODE REVIEW

**Comment by pxeger** [comment]
I think we should add a separate CI check to ensure this file (and the others generated from `elements.yaml`) get regenerated? I think that could be done with https://pre-commit.ci pretty easily.

But this is a good step for now

**Review by pxeger — APPROVED** [review]

**Comment by pxeger** [comment]
No squash please

**Comment by cgccuser** [comment]
Why no squash?

**Comment by pxeger** [comment]
~~Badminton's just better~~

Because it obscures commit history which carries useful information.

**Review by pxeger — CHANGES_REQUESTED** [review]
This needs to install `yaml` first

**Review by Lyxal — APPROVED** [review]
Maybe also add something later for documentation rerolling

**Comment by cgccuser** [comment]
Sounds good, although generating documentation should go in its own workflow.

File: .github/workflows/run-tests.yaml [hunk 1]
```diff
@@ -28,6 +28,12 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install poetry
```
Comment (pxeger) [inline .github/workflows/run-tests.yaml:29]:
```suggestion
        python -m pip install poetry yaml
```

**Review by pxeger — CHANGES_REQUESTED** [review]

**Comment by cgccuser** [comment]
There are so many commits that at this point we might as well squash them or make a new branch and add those in manually.

**Review by pxeger — APPROVED** [review]

**Comment by pxeger** [comment]
I still don't like squashing, but whatever

**Comment by cgccuser** [comment]
I mean, we could make a new branch and add those commits in again.

**Comment by cgccuser** [comment]
You know what, I'm too lazy to do that, so here we go. And fwiw, tennis > badminton.

**Pull request merged**